File: codes/funcionarioPy/vendedor.py
import funcionario as f

class Vendedor(f.Funcionario):
    def __init__(self, nome, cpf, comissao):
        super().__init__(nome, cpf)
        self.comissao = comissao

    def calculaSalario(self):
        return self.salarioBase + self.comissao

File: codes/funcionarioJava/Vendedor.java
package ex6;

public class Vendedor extends Funcionario {
    private double comissao;

    public Vendedor(String nome, String cpf, double comissao) {
        super(nome, cpf);
        this.comissao = comissao;
    }

    public double calculaSalario() {
        return salarioBase + comissao;
    }
}